app/services/experiment_control_service.rb

```diff
@@ -67,11 +58,64 @@ def start_stale_patient_experiment(name, days_til_start, days_til_end, patients_
         schedule_date += 1.day
       end
 
-      experiment.update!(state: "running")
+      experiment.running_state!
+    end
+
+    # This is not a scientific experiment like active and stale patient experiments. It does not need or have human study
+    # approval and we are not excluding patients under 18 years old. We are leveraging the experimentation framework to
+    # request patients to visit clinics to pick up enough blood pressure medication to last the current covid spike.
+    # We're using the experimentation framework for two reasons:
+    # 1) we want most of the same functionality, including batched daily notification scheduling and the ability to track
+    #    results by leveraging the experimentation and notification models
+    # 2) this is an excellent opportunity to real-world test the experimentation framework
+    def start_medication_reminder_experiment(name, patients_per_day: PATIENTS_PER_DAY)
+      return unless Flipper.enabled?(:live_experiment)
+
+      experiment = Experimentation::Experiment.find_by!(name: name, experiment_type: "medication_reminder")
+      experiment.selecting_state!
+
+      eligible_ids = medication_reminder_patients(experiment)
+      if eligible_ids.any?
+        eligible_ids.shuffle!
+
+        daily_ids = eligible_ids.pop(patients_per_day)
```

Comment:
I was thinking about maybe adding some logging here and to the other experiment controls. Perhaps log out which ids are being scheduled for the day. And maybe log out "no ids found" if there are no eligible ids for the day.